hw2/ycchan5566/q2/pybind_vector.cpp

```diff
@@ -0,0 +1,23 @@
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <iostream>
+#include <vector>
+#include <cmath>
+
+namespace py = pybind11;
+
+int angle(std::vector<int> &a, std::vector<int> &b) {
```

Comment:
you should also allow std::vector<float> std::vector<double>
I would suggest to use template here.